--- cpu_usage.py	2024-12-14 23:46:44
+++ cpu_usage_fixed.py	2024-12-14 23:48:16
@@ -4,8 +4,8 @@
 
 def check_cpu_usage(percent):
 
-    usage = psutil.cpu_percent()
-
+    usage = psutil.cpu_percent(1)
+    print("DEBUG: usage: {}".format(usage))
     return usage < percent
 
 if not check_cpu_usage(75):
@@ -15,4 +15,5 @@
 else:
 
 
-    print("Everything ok")
\ No newline at end of file
+    print("Everything ok")
+
